types/uuid.go

```diff
@@ -24,7 +24,6 @@ import (
 )
```

Comment:
Changes in this file seems unrelated to me.

Reply:
UUID is used in the `nearCacheRecord`. Having the `text` of the UUID would increase the memory usage a lot. So I've removed it.